perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/steps/SummaryStep.java

```diff
@@ -862,6 +862,14 @@ public void onClick(ClickEvent event) {
 				modal.setWidth("750px");
 
 
+				Button buttonContAnyway = new Button(translation.continueAnyway());
+				buttonContAnyway.setType(ButtonType.PRIMARY);
```

Comment:
I probably wouldn't use PRIMARY button type, since this is sort of override for the default behavior, which is to wait for the approval. Now it looks like two very similar buttons.

Or if you think it will be better this way, switch what action is the default (both button type and their order in the modal window).

Reply:
Swithched order and used `DEFAULT` for the "close but do not continue" button.